web

```diff
@@ -1 +0,0 @@
-../traccar-web/web
```

Comment:
You can use different config file with full path to web interface. Link works on Linux and Mac. Unfortunately I don't have a Windows machine to test it.